components/ProgressView/BUILD

```diff
@@ -59,9 +67,13 @@ mdc_objc_library(
 mdc_objc_library(
     name = "unit_test_sources",
     testonly = 1,
+    hdrs = native.glob([
+        "tests/unit/*.h",
+        "tests/unit/Theming/*.h",
+    ]),
     srcs = native.glob([
         "tests/unit/*.m",
-        "tests/unit/*.h",
```

Comment:
If there are currently headers being used for the tests, they should remain in the `srcs` property.

Reply:
I never thought about that really but it makes sense. I initially had some headers but got rid of them.